core/lib/Thelia/Action/Product.php

```diff
@@ -667,6 +667,7 @@ public function updateFeatureProductValue(FeatureProductUpdateEvent $event)
             // If it's a free text value, create a FeatureAv to handle i18n
             if ($event->getIsTextValue() === true) {
                 $featureProduct->setFreeTextValue(true);
```

Comment:
To prevent the use of deprecated methods in Thelia code, I suggest that `setIsFreeText() ` call `setFreeTextValue()`